src/app/exercise/exercise.page/components/exercise-settings.page/exercise-settings.page.html

```diff
@@ -44,7 +44,12 @@
       <ion-label>Adaptive</ion-label>
       <ion-toggle
         [formControl]="generalFormGroup.controls.adaptive"
-        slot="end"
+      ></ion-toggle>
+    </ion-item>
+    <ion-item>
+      <ion-label>Show answer when wrong</ion-label>
+      <ion-toggle
+        [formControl]="generalFormGroup.controls.showAswerWhenWrong"
```

Comment:
Rename variable accordingly. Most IDEs will enable you to do that refactoring automatically.